docs/DEBUGGING.md

```diff
@@ -173,38 +210,52 @@ When the start of a service fails, systemctl will give you a generic error messa
-    Active: failed (Result: exit-code) since Fri, 11 May 2012 20:26:23 +0200; 4s ago
-    Process: 1329 ExecStart=/usr/local/bin/foo (code=exited, status=1/FAILURE)
-    CGroup: name=systemd:/system/foo.service
+Loaded: loaded (/etc/systemd/system/foo.service; static)
+Active: failed (Result: exit-code) since Fri, 11 May 2012 20:26:23 +0200; 4s ago
+Process: 1329 ExecStart=/usr/local/bin/foo (code=exited, status=1/FAILURE)
+CGroup: name=systemd:/system/foo.service
 
 May 11 20:26:23 scratch foo[1329]: Failed to parse config
 ```
 
-
-In this example the service ran as a process with PID 1329 and exited with error code 1. If you run systemctl status as root or as a user from the `adm` group, you will get a few lines from the journal that the service wrote. In the example the service produced just one error message.
+In this example the service ran as a process with PID 1329 and exited with error code 1.
+If you run systemctl status as root or as a user from the `adm` group,
+you will get a few lines from the journal that the service wrote.
+In the example the service produced just one error message.
 
 To list the journal, use the `journalctl` command.
 
-If you have a syslog service (such as rsyslog) running, the journal will also forward the messages to it, so you'll find them in **/var/log/messages** (depending on rsyslog's configuration).
-
+If you have a syslog service (such as rsyslog) running, the journal will also forward the messages to it,
+so you'll find them in **/var/log/messages** (depending on rsyslog's configuration).
 
 # Reporting systemd Bugs
 
-Be prepared to include some information (logs) about your system as well. These should be complete (no snippets please), not in an archive, uncompressed.
+Be prepared to include some information (logs) about your system as well.
+These should be complete (no snippets please), not in an archive, uncompressed.
 
-Please report bugs to your distribution's bug tracker first. If you are sure that you are encountering an upstream bug, then first check [for existing bug reports](https://github.com/systemd/systemd/issues/), and if your issue is not listed [file a new bug](https://github.com/systemd/systemd/issues/new).
+Please report bugs to your distribution's bug tracker first.
+If you are sure that you are encountering an upstream bug, then first check
+[for existing bug reports](https://github.com/systemd/systemd/issues/),
+and if your issue is not listed
+[file a new bug](https://github.com/systemd/systemd/issues/new).
 
 ## Information to Attach to a Bug Report
 
 Whenever possible, the following should be mentioned and attached to your bug report:
 
-* The exact kernel command-line used. Typically from the bootloader configuration file (e.g. **/boot/grub2/grub.cfg**) or from **/proc/cmdline**
+* The exact kernel command-line used.
+Typically from the bootloader configuration file (e.g. **/boot/grub2/grub.cfg**) or from **/proc/cmdline**
```

Comment:
Backticks for the path, instead of bolding?

Reply:
We didn't want to change the existing text format.